src/components/SnapshotDetails/tabs/SnapshotPipelineRunsList.tsx

```diff
@@ -119,7 +120,7 @@ const SnapshotPipelineRunsList: React.FC<React.PropsWithChildren<SnapshotPipelin
         className="pf-v5-c-title pf-v5-u-mt-lg pf-v5-u-mb-lg"
         data-test="snapshot-plr-title"
       >
-        Pipeline runs
+        Pipeline runs <FeatureFlagIndicator flags={['pipelineruns-kubearchive']} fullLabel />
```

Comment:
nit: I think for these features we're not displaying the flag with "full label" :)